docsk8s/troubleshooting/ls-k8s-troubleshooting-methods.asciidoc

```diff
@@ -1,35 +1,95 @@
 [[ls-k8s-troubleshooting-methods]]
 === Troubleshooting methods
 
-There are a number of methods that you can use to diagnose the state of your Logstash and Kubernetes system, both in the event of any problems, and as part of a day-to-day approach to ensuring that everything is running as expected.
+There are a number of methods that you can use to diagnose the state of your {ls} and Kubernetes system, both in the event of any problems, and as part of a day-to-day approach to ensuring that everything is running as expected.
 
-<<ls-k8s-checking-resources>>
-<<ls-k8s-viewing-logs>>
-<<ls-k8s-connecting-to-a-container>>
+* <<ls-k8s-checking-resources>>
+* <<ls-k8s-viewing-logs>>
+* <<ls-k8s-connecting-to-a-container>>
+* <<ls-k8s-diagnostics>>
 
 [float]
 [[ls-k8s-checking-resources]]
 === Checking resources
 
-You can use the standard Kubernetes `get` and `describe` commands to quickly gather details about any resources in your Logstash and Kubernetes environment.
+You can use the standard Kubernetes `get` and `describe` commands to quickly gather details about any resources in your {ls} and Kubernetes environment.
 
-Examples...
+[source,bash]
+--
+kubectl get pod logstash-7477d46bb7-4lcnv
 
+NAME                                    READY   STATUS        RESTARTS         AGE
+logstash-7477d46bb7-4lcnv               0/1     Pending       0                2m43s
+--
+
+If a Pod fails to reach the `Running` status after a few seconds, run this command to get more insights:
```

Comment:
Nice.  The "run this command" construct is much better than using "the following."  Those are extra works that don't add any value, so it's nice that you left them out.